<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibManager</class>
 <widget class="QWidget" name="LibManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(98, 93, 255);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="logintitle">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;华文行楷&quot;;</string>
          </property>
          <property name="text">
           <string>词库管理</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>38</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;关闭&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
	
	border-image: url(:/searchwindow/searchwindow/close_normal.png);
}

QToolButton:hover{
	
	border-image: url(:/searchwindow/searchwindow/close_hover.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>词典选择</string>
     </property>
     <widget class="QRadioButton" name="WeiKeiBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>维科英汉词典</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="ClanguageBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C语言库函数</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="SimilarBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>英语同义词辨析</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="ProverbBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>英语谚语</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="HuaWeiBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>华为网络通信术语</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="PoemsBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>唐诗300首</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="EngineerBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>电子工程词典</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="ChineseBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>中华大辞典</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quedingBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
